### PR TITLE
Update usage documentation

### LINKED ISSUE #23: Document the VS and terminal flows
We have two main supported flows:

1. Visual Studio integration: user loads the project, invokes `conan install` through the plugin, and then the project should build successfully
2. Command line integration (e.g. for build servers etc.): something like `cd debug64 && conan install .. && cd .. && msbuild`

Document that.

### PR DESCRIPTION
**Pull request by ForNeVeR**
Closes #23, and officially closes our first milestone. I'll show the results to bincrafters and we'll discuss what to do next.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,31 +10,66 @@ provides for .NET projects.
...
+command. It will automatically add the corresponding `<Import>` item into the
+`vcxproj` file.
 
-### Using the tool window
+!["Integrate into project" menu item screenshot][screenshot-integrate]
+
+### Building without Visual Studio
+
+If you need to build the generated project without Visual Studio (e.g. on the
+build server machine), execute the following terminal commands:
```
Comment (Minoru) [inline README.md]: I'd write "a build server", because you're talking of *some* build server, not a specific one.

File: README.md [hunk 2]
```diff
@@ -10,31 +10,66 @@ provides for .NET projects.
...
+After calling `conan install` that way, `msbuild` will be able to find all the
+dependencies, because it'll be able to use the Conan-generated
+`conan/conanbuildinfo.props` file.
 
 Build
 -----
 
-To build the project, either build it using Visual Studio or with MSBuild:
+To build the Conan Visual Studio plugin, either build it using Visual Studio or
+with MSBuild:
```
Comment (Minoru) [inline README.md]: I see three problems with this sentence:
1. "To build… build" is a tautology;
2. "using Visual Studio" but "with MSBuild";
3. not clear how to build with Visual Studio.

It seems like by "build using Visual Studio" you mean the recipe from "Usage" section above, but I'm not totally sure. I'm also not sure why the `msbuild` invocation in this section is different from the one in "Building without Visual Studio" section.

I don't know enough to suggest an alternative, but can't this be rewritten somehow?

Comment (ForNeVeR) [reply README.md]: > It seems like by "build using Visual Studio" you mean the recipe from "Usage" section above, but I'm not totally sure. I'm also not sure why the msbuild invocation in this section is different from the one in "Building without Visual Studio" section.

This document describes two separate processes:

- using the plugin (to build **your** software)
- building the plugin itself

I'll look how to rewrite that, thanks.

**Comment by ForNeVeR** [comment]
Thanks for the review, guys!

**Review by ttldtor — APPROVED** [review]

**Pull request merged**